R/bundle.R

```diff
@@ -834,7 +834,7 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
 
 validatePackageSource <- function(pkg) {
   msg <- NULL
-  if (!(pkg$Source %in% c("CRAN", "Bioconductor", "github"))) {
+  if (!(pkg$Source %in% c("CRAN", "Bioconductor", "github", "gitlab"))) {
```

Comment:
It is possible that `bitbucket` is missing here too - Does bitbucket packages works ? 🤔

Reply:
I think that you're right. It looks like it needs all the SCM types plus CRAN and Bioconductor.

Reply:
Would you mind adding a NEWS entry?

Reply:
This definitely needs `bitbucket`. We are not producing an error here because `pkg$Repository` is `NA` rather than `NULL`. Let's not change that check right now, however, as we're close to a CRAN release. It would be good to have bitbucket added, though.

CC @ronblum

Reply:
Thanks @cderv. This gives me:

```r
Error in http(service$protocol, service$host, service$port, method, url,  : 
  object 'jsonTracingEnabled' not found
Calls: <Anonymous> ... grepl -> GET -> httpRequest -> httpInvokeRequest -> http
Execution halted
```

Tested on both local IDE and RStudio server pro.

Reply:
@kevinushey - could this be fallout from https://github.com/rstudio/rsconnect/pull/490?

Reply:
@aronatkins . In the meantime I restarted from @cderv branch and commented out the `jsonTracingEnabled` code, to see whether I can go further.

Reply:
FWIW this specific issue about `jsonTracingEnabled `is in one of the failed test: https://travis-ci.com/github/rstudio/rsconnect/builds/221234756#L6653

Reply:
Sorry, just fixed that here: https://github.com/rstudio/rsconnect/commit/87faf044e3414edc5828ad6ec33d7e8d1488dd1b

Reply:
@aronatkins I added bitbucket